src/metacognition/confidence_estimator.py

```
@@ -0,0 +1,466 @@
+            weighted_sum += score * weight
+            total_weight += weight
+
+        if total_weight == 0:
+            return 0.5
+
+        return weighted_sum / total_weight
+
+    def _generate_help_request(
+        self,
+        factors: Dict[ConfidenceFactor, float],
+        uncertain_aspects: List[str],
```

Ruff (0.14.10)

Unused method argument: `uncertain_aspects`

(ARG002)